internal/storage/storage.go

```diff
@@ -70,7 +75,20 @@ type VehicleRepository interface {
 }
 
 type TreeClusterRepository interface {
-	BasicCrudRepository[entities.TreeCluster]
+	// GetAll returns all tree clusters
```

Comment:
I think the comments are a bit out of place here. I think the comments are better placed where the function is also created, so that you can also specify parameters and return values if required. Otherwise this file will get messy.

Reply:
Yes, but the problem is that the service uses this interface, if you specify the comment in the implementation, the LSP will not register it because it has no idea of the implementation, only the interface. So it has to be in the interface definitions

Reply:
I do not get it. What is a LSP?

Reply:
LSP stands for [`Language Server Protocol`](https://microsoft.github.io/language-server-protocol/). If I comment the function, it should be visible in the function definition. But because we are using the interface in the service layer, only these comments are visible, not the comments in the functions.

**Comment on interface visible in the service**
![image](https://github.com/user-attachments/assets/e523e720-43fb-4a0f-93c5-717c509f901a)

**Comment on the funtion not visible in the service**
![image](https://github.com/user-attachments/assets/05539b3f-674f-4443-8fcc-45180fad0cab)